src/Ocelot/Configuration/File/FileGlobalConfiguration.cs

```diff
@@ -17,6 +17,8 @@ public FileGlobalConfiguration()
 
         public string RequestIdKey { get; set; }
 
+        public int RequestTimeoutSeconds { get; set; } = 90;
```

Comment:
Wrong intention to encapsulate some value which is not shared.
We usually define public constant! See examples: [search for "90"](https://github.com/search?q=repo%3AThreeMammals%2FOcelot+90+language%3AC%23&type=code&l=C%23)

### Returning type must be `Nullable`!

Reply:
fixed